ui/components/app/edit-gas-fee-popover/network-statistics/status-slider/status-slider.test.js

```diff
@@ -20,31 +20,31 @@ const renderComponent = ({ networkCongestion }) => {
 
 describe('StatusSlider', () => {
   it('should show "Not busy" when networkCongestion is less than 0.33', () => {
-    const { queryByText } = renderComponent({ networkCongestion: 0.32 });
```

Comment:
Apparently I forgot the difference between `query*` and `get*` 😆  But at the same time all of these tests change because the definition changed.